spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java

```diff
@@ -454,6 +454,8 @@ public void testQueryMatching() {
     assertThat(SparkBigQueryConfig.isQuery("SELECT\ta,b from table")).isTrue();
     assertThat(SparkBigQueryConfig.isQuery("WITH bar AS (SELECT * FROM foo)\nSELECT * FROM bar"))
         .isTrue();
+    assertThat(SparkBigQueryConfig.isQuery("select--comment\n* from table")).isTrue();
+    assertThat(SparkBigQueryConfig.isQuery("select col1 -- comment\nfrom table")).isTrue();
```

Comment:
Question: what about the case `select col1 from table -- comment`? is this case supported?

Reply:
Although this currently works, it is a good case to handle. Thanks!